README.md

```diff
@@ -17,8 +17,8 @@ For more information about how tests are currently handled, see this [repo](http
 
 To add the Integration Test Scenario to Konflux, you need to follow these next steps:
 
-1. Fork the tenants-config [repo](https://github.com/redhat-appstudio/tenants-config.git)
-2. In case there is still not a directory for your tenant in the repo you just forked, you will need to create one under the directory `cluster/stone-prd-rh01/tenants/<your-workspace-name>-tenant`.
+1. Fork the konflux-release-data [repo](https://gitlab.cee.redhat.com/releng/konflux-release-data.git)
+2. In case there is still not a directory for your tenant in the repo you just forked, you will need to create one under the directory `tenants-config/cluster/stone-prd-rh01/tenants/<your-workspace-name>-tenant`.
```

Comment:
Change `stone-prd-rh01` for `<your-cluster>` because it will depend on which cluster you are using. You can add a link to https://gitlab.cee.redhat.com/konflux/docs/users/-/blob/main/topics/overview/deployments.md?ref_type=heads to know which cluster you are using